### PR TITLE
Add rubocop

### PR DESCRIPTION
**Pull request by kashifatcha**
## What

Add rubocop-govuk to [pay-pagerduty](https://github.com/alphagov/pay-pagerduty), following the same conventions as other [GOV.UK](http://gov.uk/) repos.

Ensure that it is also run as part of CI.

## Why

To simplify code reviews and avoid reinventing the wheel, and arrive at consistent code style.

Note that despite the repo name, this is used and currently owned by [GOV.UK](http://gov.uk/). A name change is likely in future!

Other info

This is non-urgent work to be completed by 2nd line (only if other priorities allow) to improve our long term processes and health. Please seek code review from Chris Ashton or Deborah Chua before merging changes.

https://trello.com/c/s8vlPmqy/1967-add-linting-to-pay-pagerduty

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  rubocop-govuk:
+    - config/default.yml
+    - config/rspec.yml
+
+inherit_mode:
+  merge:
+    - Exclude
+
+Style/DateTime:
+  Enabled: false
```
Comment (ChrisBAshton) [inline .rubocop.yml]: Could you add an inline comment explaining why this is disabled?
I see "Here I have disable Style/DateTime as it wasn't a safe autocorrect." in the commit message but I'm not clear on why it's unsafe?

File: spec/overrider_spec.rb [hunk 2]
```diff
@@ -52,7 +52,7 @@
     ])
   end
 
-  context "a rotation which overlaps the beginning of the UTC/BST transition" do
```
Comment (ChrisBAshton) [inline spec/overrider_spec.rb:55]: Nice job fixing these up 👍

**Review by ChrisBAshton — COMMENTED** [review]
Looks good to me - nice job, @kashifatcha! ⭐ 

One nitpick comment below, and also a general comment that we should start all commit messages with a capital letter (please fix up and rebase).

File: .rubocop.yml (hunk 1 above)
Comment (kashifatcha) [reply .rubocop.yml]: I have added to the commit message with further explanation :)

Comment (kashifatcha) [reply .rubocop.yml]: I've also added an inline comment, sorry for the delay!

**Review by ChrisBAshton — APPROVED** [review]
Looks good - thanks @kashifatcha !

**Pull request merged**